gulpfile.js

```diff
@@ -16,6 +16,12 @@ gulp.task('copy-static', function () {
     .pipe(gulp.dest('./application/static'))
 })
 
+// Copy assets from GOV.UK Frontend
+gulp.task('copy-assets', function () {
```

Comment:
Maybe `copy-govuk-assets` in case we have others at some point?

Reply:
👍  have renamed this `copy-govuk-frontend-assets` to be super specific.